<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestVideoSetting</class>
 <widget class="QDialog" name="TestVideoSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>视频设置</string>
  </property>
  <widget class="QWidget" name="localVideoView" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>视频帧率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="videoBitrate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bitrate 550kbps：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="resMode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率模式：</string>
   </property>
  </widget>
  <widget class="QSlider" name="bitrateSlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>500</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>550</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="resComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>横屏分辨率</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>竖屏分辨率</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="fpsComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>15fps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20fps</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="camComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="videoResolution">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string>640_360</string>
   </property>
   <property name="currentIndex">
    <number>16</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>120_120   // 宽高比1:1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160_160</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>270_270</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480_480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160_120   // 宽高比4:3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240_180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>280_210</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>320_240</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>400_300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480_360</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>640_480</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>960_720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160_90     // 宽高比16:9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256_144</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>320_180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480_270</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>640_360</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>960_540</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280_720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920_1080</string>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="whitenessLevelSlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>710</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="playoutVolumeLabel_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>750</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>红润级别 0 - 9：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="beautyStyleComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>630</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>光滑</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自然</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="playoutVolumeLabel_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>美颜风格：</string>
   </property>
  </widget>
  <widget class="QSlider" name="beautyLevelSlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>670</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="playoutVolumeLabel_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>670</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>磨皮级别 0 - 9：</string>
   </property>
  </widget>
  <widget class="QLabel" name="playoutVolumeLabel_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>710</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>美白级别 0 - 9：</string>
   </property>
  </widget>
  <widget class="QSlider" name="ruddinessLevelSlider">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>750</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="remoteFillModeComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>填充</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>适应</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="localFillModeComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>填充</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>适应</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>本地填充模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>远端填充模式：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="setWaterMarkBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置 / 取消水印</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>421</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <zorder>listView_3</zorder>
  <zorder>listView_2</zorder>
  <zorder>localVideoView</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>videoBitrate</zorder>
  <zorder>resMode</zorder>
  <zorder>bitrateSlider</zorder>
  <zorder>resComboBox</zorder>
  <zorder>fpsComboBox</zorder>
  <zorder>camComboBox</zorder>
  <zorder>videoResolution</zorder>
  <zorder>whitenessLevelSlider</zorder>
  <zorder>playoutVolumeLabel_5</zorder>
  <zorder>beautyStyleComboBox</zorder>
  <zorder>playoutVolumeLabel_4</zorder>
  <zorder>beautyLevelSlider</zorder>
  <zorder>playoutVolumeLabel_2</zorder>
  <zorder>playoutVolumeLabel_3</zorder>
  <zorder>ruddinessLevelSlider</zorder>
  <zorder>remoteFillModeComboBox</zorder>
  <zorder>localFillModeComboBox</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>setWaterMarkBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
